src/index.html

```diff
@@ -11,6 +11,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="searchbar__form" data-qa="big">
+      <label class="searchbar__label searchbar__label--big">
+        <input
+        type="text"
+        class="searchbar__input searchbar__input--big"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+      >
```

Comment:
Pay attention on indents
```suggestion
        <input
          type="text"
          class="searchbar__input searchbar__input--big"
          placeholder="Try “Los Angeles”"
          data-qa="keypress"
        >
```